hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/record/RecordListBuilderTest.java

```diff
@@ -645,6 +646,43 @@ void testAddMultipleRemovableChildren() {
                 .hasParent(result.userTransactionRecord());
     }
 
+    @Test
+    void testAddRemovableChildWithSuppressedRecord() {
+        // given
+        final var consensusTime = Instant.now();
+        final var recordListBuilder = new RecordListBuilder(consensusTime);
+        addUserTransaction(recordListBuilder);
+
+        final var suppressionCustomizer = new ExternalizedRecordCustomizer() {
+            @Override
+            public Transaction apply(Transaction transaction) {
+                throw new UnsupportedOperationException("The top-level creation record should be suppressed");
+            }
+
+            @Override
+            public boolean shouldSuppressRecord() {
+                return true;
+            }
+        };
```

Comment:
You could now use `SUPPRESSING_EXTERNALIZED_RECORD_CUSTOMIZER` here (which of course works because you gave it its own test)